gradle-plugin/src/main/java/com/squareup/anvil/plugin/AnvilPlugin.kt

```diff
@@ -71,15 +72,15 @@ open class AnvilPlugin : KotlinCompilerPluginSupportPlugin {
         ),
         SubpluginOption(
           key = "generate-dagger-factories",
-          value = extension.generateDaggerFactories.toString()
+          lazy { extension.generateDaggerFactories.getOrElse(false).toString() }
```

Comment:
This probably comes down to style, but I just want to be sure you know that an alternative to `getOrElse(false)` would be to have `convention(false)` when you instantiate those properties. I think it's a matter of which thing you want to have to remember 🙃